packages/hop-node/src/watchers/classes/BaseWatcherWithEventHandlers.ts

```diff
@@ -122,33 +122,36 @@ class BaseWatcherWithEventHandlers extends BaseWatcher {
   }
 
   async handleTransferRootBondedEvent (
-    transferRootHash: string,
-    totalAmount: BigNumber,
+    root: string,
+    amount: BigNumber,
     event: Event
   ) {
-    const logger = this.logger.create({ root: transferRootHash })
+    const logger = this.logger.create({ root: root })
     logger.debug('handling TransferRootBonded event')
 
     try {
       const { transactionHash } = event
       const tx = await this.bridge.getTransaction(transactionHash)
       const { from: bonder } = tx
+      const eventTimestamp = await this.bridge.getEventTimestamp(event)
       const transferRootId = await this.bridge.getTransferRootId(
-        transferRootHash,
-        totalAmount
+        root,
+        amount
       )
 
-      logger.debug(`transferRootHash from event: ${transferRootHash}`)
-      logger.debug(`bondAmount: ${this.bridge.formatUnits(totalAmount)}`)
+      logger.debug(`transferRootHash from event: ${root}`)
+      logger.debug(`bondAmount: ${this.bridge.formatUnits(amount)}`)
       logger.debug(`transferRootId: ${transferRootId}`)
       logger.debug(`event transactionHash: ${transactionHash}`)
       logger.debug(`bonder: ${bonder}`)
 
-      await this.db.transferRoots.update(transferRootHash, {
-        transferRootHash,
+      await this.db.transferRoots.update(root, {
+        transferRootHash: root,
         transferRootId,
-        committed: true,
+        confirmed: true,
```

Comment:
You are correct. `confirmed` and `confirmedAt` should be removed.

1. I believe `committed` should still be removed here, since `committed` should be handled in `checkTransfersCommitted`. In the case where this is truly fraudulent, the root will have never _actually_ been committed. Do you agree?
2. Removing `confirmed` and `confirmedAt` will require a change in the `getChallengeableTransferRoots()` function. This makes sense. I believe `item.confirmed` and `item.ConfirmedAt` should simply be replaced with `item.bonded`, as that makes the most logical sense. Do you agree?

Reply:
1. I think `committed: true` should only be set on the transfersComitted event, so it should be removed [here](https://github.com/hop-protocol/hop/blob/16529b7854b6dd736a48e67f014413ef291f0152/packages/hop-node/src/watchers/classes/BaseWatcherWithEventHandlers.ts#L150) and then the db getter should have `item.committed`

Reply:
Are you still planning on removing confirmed and confirmedAt here? since the bonded root doesn't always imply the root is confirmed